pkg/server/version.go

```diff
@@ -17,29 +17,30 @@ limitations under the License.
 package server
 
 import (
-	"fmt"
-
 	"golang.org/x/net/context"
 	"k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime"
+
+	"github.com/kubernetes-incubator/cri-containerd/pkg/version"
 )
 
 const (
-	containerName        = "containerd"
+	// For now, containerd and runc are bundled with cri-containerd, cri-containerd
+	// version is more important to us.
+	// TODO(random-liu): Figure out how to package cri-containerd and containerd,
+	// and how to version it. We still prefer calling the container runtime "containerd",
+	// but we care both the cri-containerd version and containerd version.
+	containerName        = "cri-containerd"
 	containerdAPIVersion = "0.0.0"
 	// kubeAPIVersion is the api version of kubernetes.
 	kubeAPIVersion = "0.1.0"
 )
 
 // Version returns the runtime name, runtime version and runtime API version.
 func (c *criContainerdService) Version(ctx context.Context, r *runtime.VersionRequest) (*runtime.VersionResponse, error) {
-	resp, err := c.client.Version(ctx)
```

Comment:
hate hiding useful information... maybe fix this at the CRI level with an array of versions.. dunno

Reply:
Yeah, that's why I filed the TODO. We should expose both containerd and cri-containerd verison if we don't bundle them together, but there is no way to do that in current CRI.

For our current testing, we need cri-containerd version more, so we use cri-containerd version for now. But we should find a way to fix this.

This is a cri-containerd specific problem, because we have 2 daemons out of kubelet. Both dockershim and cri-o don't have this problem, because they either built inside kubelet (dockershim) or only have one daemon (cri-o).